Website/src/app/components/layout_base.tsx

```diff
@@ -0,0 +1,17 @@
+import '../../components/globals.css'
+import Header from "@/app/components/header";
+import Footer from "@/app/components/footer";
+
+export default function Layout({
+    children,
+  }: {
+    children: React.ReactNode
+  }) {
+    return (
+        <div className='h-full min-h-screen flex flex-initial flex-col'>
```

Comment:
To organize the styling between the different files all the stylings should be structured into .css files and only referenced by their class here.

Reply:
WON'T FIX: this sort of styling is only used for this component. There is absolutely nothing gained by defining a custom CSS class `page_container` that magically applies the same style. Additionally, I would then have to look at two different files to figure out why something is styled like it is. (Also it would defy the use of tailwind for this page)

I would argue, that it also helps with maintainability, as there are no 'orphaned' CSS styles hanging around in a CSS file somewhere, that might still be in use

However, (not in this case, but maybe for some other thing) I should move some duplicated layout things into a component or a `layout.tsx` file.